cmd/root.go

```diff
@@ -34,7 +34,7 @@ to quickly create a Cobra application.`,
 			os.Exit(1)
 		}
 
-		root, err := dox.Publish("", "", dryRun) // generated root page
+		root, err := dox.Publish("", "", repoRoot, dryRun) // generated root page
```

Comment:
Where is `repoRoot` defined?

Reply:
Declared here: https://github.com/jesselang/dox/blob/master/cmd/root.go#L15
Defined here: https://github.com/jesselang/dox/blob/master/cmd/root.go#L74

Reply:
Of course! That was a silly question. Lol